substrate/frame/democracy/src/migrations/v1.rs

```diff
@@ -18,7 +18,13 @@
 //! Storage migrations for the preimage pallet.
 
 use crate::*;
-use frame_support::{pallet_prelude::*, storage_alias, traits::OnRuntimeUpgrade, BoundedVec};
+use frame_support::{
+	// We cannot use the `pallet_prelude` here as the item `Bounded` will become ambiguous.
+	pallet_prelude::{StorageVersion, ValueQuery},
```

Comment:
True, because we removed `Bounded` from `frame_support::pallet_prelude::*`. Reverted in https://github.com/paritytech/polkadot-sdk/pull/5995/commits/d46029526908a15a9d76c738afc92168643cea1b

Reply:
This is a good example of where adding more things to frame_support::pallet_prelude::* turned out to be a headache.